### DISCUSSION TITLE
Audio Ubuntu,debian etc

### DISCUSSION THREAD

**Discussion by MusicaFM** [termux/termux-app#2756] [Q&A]
Hola. ¿Audio en Ubuntu,Debian,etc desde ssh sin app de terceros? Sonido desde ssh como ocurre desde el sistema operativo de Termux
Gracias

**Comment by zxrovx** [comment]
> 

use pulseaudio, para conectar el sonido de ubuntu en una aplicación de terceros con termux

**Reply by MusicaFM** [reply]
Ya lo conseguí sin usar una app de terceros, el sonido sale desde el SO termux hacia Ubuntu Termux, más info usando Google
